build.cmd

```diff
@@ -80,6 +82,11 @@ for %%T IN (Restore %MSBuildBuildTarget%, BuildModernVsixPackages) do (
   )
 )
 
+REM Run copy as a final step after all the product components are built
+if /I "%CopyOutputArtifacts%" == "true" (
+  call build\Scripts\CopyOutput.cmd
```

Comment:
Do you mean something like this? 

```
if /I "%CopyOutputArtifacts%" == "true" (
  call build\Scripts\CopyOutput.cmd
  || goto :BuildFailed
)
```

meaning if copy fails then build fails too?